packages/sdk/src/dataProtector/sharing/createCollection.ts

```diff
@@ -0,0 +1,42 @@
+import { ethers, type Contract } from 'ethers';
+import { ABI as sharingABI } from '../../contracts/sharingAbi.js';
+import { WorkflowError } from '../../utils/errors.js';
+import { throwIfMissing } from '../../utils/validators.js';
+import { AddressOrENS, IExecConsumer } from '../types.js';
+
+export type CreateCollectionResponse = {
+  collectionId: AddressOrENS;
+};
```

Comment:
Trying something here to slow down `types.ts` file length: the method file will own its types as we can see here.